### DISCUSSION TITLE
battery models not supported

### DISCUSSION THREAD

**Discussion by alexmogavero** [fwestenberg/reolink_dev#233] [Q&A]
Why models powered on batteries are not supported. From reolink documentations it seems they share the same software interface of AC powered ones

**Comment by mttyd** [comment]
Looks like a lot of dead ends on this one.  I was thinking about the Google integration but it looks like you can only ask Google to show you the camera on a chromecast device (not Home assistant).

**Reply by alexmogavero** [reply]
At a closer look to reolink documentations it seems battery powered cameras have a lot of caveats. Namely due to power saving issues. I have chosen to move away from battery powered cameras too many limitations.
